.github/workflows/ci.yml

```diff
@@ -73,12 +71,11 @@ jobs:
     - uses: actions/checkout@v3
       with:
         ref: ${{ github.head_ref }}
-        token: ${{ secrets.GitHubToken }}
```

Comment:
If the update-action below opens a PR or otherwise pushes code, it probably needs this different token than the default because by default the token given to jobs does not have that capability.

Reply:
so to be honest I don't really know why this was here. It was complaining before that there was no such token (maybe last time it passed the secrets object wasn't case sensitive because there is a `GITHUBTOKEN` configured 🤔)

But I have had this pass without the token being passed and it just logs "not a blueprint" so 🤷  https://github.com/ember-cli/ember-cli-update/actions/runs/5993156224/job/16253317325

Reply:
I tried to add it back with the correct name `secrets.GITHUBTOKEN` but it's not working 🤔 so I'm leaving it off for now and if the CI job starts failing at any time I'll fix it there 👍

Reply:
The builtin one is `GITHUB_TOKEN`